mediapipe/tasks/ios/components/containers/sources/MPPEmbedding.m

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "mediapipe/tasks/ios/components/containers/sources/MPPEmbedding.h"
+
+@implementation MPPEmbedding
+
+- (instancetype)initWithFloatEmbedding:(nullable NSArray<NSNumber *> *)floatEmbedding
+                    quantizedEmbedding:(nullable NSData *)quantizedEmbedding
+                             headIndex:(NSInteger)headIndex
+                              headName:(nullable NSString *)headName {
+  self = [super init];
+  if (self) {
+    _headIndex = headIndex;
```

Comment:
@priankakariatyml @schmidt-sebastian As an embedding can't be both quantized and float, should we do a check here to make sure that MPPEmbedding isn't initialized with both float and quantized embedding?

Reply:
Prianka asked me in an earlier comment. I think it is okay not to do so.